logger.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import datetime
+import config
+
+# Python has a logging library that will do what we want.
+# Basic documentation here:     https://docs.python.org/3/howto/logging.html
+# Advanced documentation here:  https://docs.python.org/3/howto/logging-cookbook.html
+
+# This should be called once in main, before any calls to the logging library
+def initialize_logging():
+    # Defines the format of the logs
+    log_format = '%(asctime)s %(name)-12s %(levelname)-8s %(message)s'
+    date_format = '%Y-%m-%d %H:%M:%S'
+    # Get current time in order to create log file name
+    timeNow = datetime.datetime.now()
+    timeStr = f"{timeNow.year}{timeNow.month}{timeNow.day}_{timeNow.hour}_{timeNow.minute}_{timeNow.second}"
+    # Set up logging to file
+    logging.basicConfig(
+        level=logging.DEBUG,
+        format=log_format,
+        datefmt=date_format,
+        filename=f"Logs/FFX_Log_{timeStr}.txt",
+        filemode='w'
+    )
+
+    # Set up the console logger, which may have different visible log level
+    config_data = config.open_config()
+    console_log_level = config_data.get("verbosity", "DEBUG")
```

Comment:
While we are in here, we probably want to always log everything into a log file, but only log stuff to the terminal based on `verbosity`

Reply:
This is the case! The `logging.basicConfig(...` code sets up the file logger, which will always use DEBUG (ie everything).

Reply:
Nice, I was unaware. I'm used to some other tool hi-jacking all the logging and I don't know what the default is and what that tool is doing ontop of the standard defaults.